frontend/src/pages/GraphicalSelector/GraphicalSelector.jsx

```diff
@@ -104,28 +106,24 @@ const GraphicalSelector = () => {
   };
 
   const setupGraph = async () => {
+    // new epic version
```

Comment:
Why not use the new epic version right away over the old one? xD Though if the BE is already done with the setup, FE can take it from here 😅

Reply:
i thought this was not possible because i didnt know how to add the extra information in and just adding the data from the new request to `initialiseGraph` didn't seem to work. I now realise I'm being silly and needed to unpack stuff out of `data` first.

Now using new version now, all good.

```js
  const setupGraph = async () => {
    const { data } = await axios.get(`/programs/graph/${programCode}/${specs.join("+")}`);
    const { edges, courses } = data;

    if (courses.length !== 0 && edges.length !== 0) initialiseGraph(courses, edges);
    setLoading(false);
  };
```